spec/functional/caching_spec.rb

```diff
@@ -43,17 +43,15 @@
   end
 
   it 'should fail when cacheable but missed' do
-    expect { expect(DOMAIN + '/cacheable_but_miss').to not_be_cached }.to raise_error(RuntimeError)
+    expect(DOMAIN + '/cacheable_but_miss').not_to not_be_cached
```

Comment:
"Not to not be cached" is a bit harder.  I wonder if you could have another matcher which just negates `not_be_cached`, so you could instead say things like:
`expect(...).to be_cached` and `expect(...).not_to be_cached`.